### PR TITLE
Frontend: Added validation for new password

### PR DESCRIPTION
**Pull request by Curious72**
I have created a new directive named "match" for matching if the old password and new passwords match and returns false if they match .

### CODE REVIEW

**Comment by Curious72** [comment]
Fixes #775

File: frontend/src/views/web/update-profile.html [hunk 1]
```diff
@@ -6,12 +6,13 @@
                 <div class="ev-md-container ev-card-body update-profile-card">
                     <form name="updateProfileForm" ng-submit="updateProfile.updateProfile(updateProfileForm.$valid)" novalidate>
                         <div class="input-field align-left">
-                            <input id="username" name="username" type="text" class="text-dark-black dark-autofill w-400"  ng-model="updateProfile.user.username" ng-maxlength="30" required focus-if>
+                            <input id="username" name="username" type="text" class="text-dark-black dark-autofill w-400"  ng-model="updateProfile.user.username" ng-maxlength="30" ng-minlength="3" required focus-if>
```
Comment (aka-jain) [inline frontend/src/views/web/update-profile.html]: @Curious72 please make these changes to separate PR.

**Comment by Curious72** [comment]
@aka-jain please review this !

**Comment by aka-jain** [comment]
LGTM.
